<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed'); ?>

ERROR - 2014-03-24 22:11:43 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'index'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('index', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:11:57 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:12:05 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:12:14 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:14:17 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:15:19 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:16:36 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:16:36 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
ERROR - 2014-03-24 22:16:42 --> exception 'Lynx_ErrorException' with message 'call_user_func_array() expects parameter 1 to be a valid callback, class 'login' does not have a method 'images'' in C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php:75
Stack trace:
#0 [internal function]: _exception_handler(2, 'call_user_func_...', 'C:\Users\ta\Doc...', 75, Array)
#1 C:\Users\ta\Documents\GitHub\e-com\src\application\core\MY_Controller.php(75): call_user_func_array(Array, Array)
#2 C:\Users\ta\Documents\GitHub\e-com\src\system\core\CodeIgniter.php(323): MY_Controller->_remap('images', Array)
#3 C:\Users\ta\Documents\GitHub\e-com\src\docroot\index.php(258): require_once('C:\Users\ta\Doc...')
#4 {main}
